pipelines/reconfigure.yml

```
@@ -38,6 +38,82 @@ resources:
       password: ((basic_auth.password))
 
 jobs:
+- name: reconfigure-resource-pipelines
+  plan:
+  - get: pipelines
+    trigger: true
+  - task: render-resource-templates
+    file: pipelines/tasks/render-resource-pipeline-templates.yaml
+    params:
+      RESOURCES: |
+        git
+        time
+        docker-image
+        registry-image
+        cf
+        bosh-io-release
+        bosh-io-stemcell
+        tracker
+        hg
+        github-release
+        semver
+        s3
+        concourse-pipeline
+        pool
+        datadog-event
+        mock
+  - put : prod
```

If I got it right, I think we don't need any other inputs other than `rendered_pipelines` 🤔  

```suggestion
  - put: prod
    inputs: [rendered_pipelines]
```

(soon we'll be able to use `detect` though 👀  https://github.com/concourse/concourse/pull/4616)

---

hahaha I originally had it this way and opted for less code! Maybe it's worth just getting in the habit